checkout/access-checkout-android/src/test/java/com/worldpay/access/checkout/validation/CVVValidatorTest.kt

```diff
@@ -1,289 +1,302 @@
 package com.worldpay.access.checkout.validation
 
-import com.worldpay.access.checkout.api.configuration.CardConfiguration
 import com.worldpay.access.checkout.testutils.CardConfigurationUtil.Brands.VISA_BRAND
 import com.worldpay.access.checkout.testutils.CardConfigurationUtil.Configurations.CARD_CONFIG_BASIC
-import com.worldpay.access.checkout.testutils.CardConfigurationUtil.Defaults.CARD_DEFAULTS
+import com.worldpay.access.checkout.testutils.CardConfigurationUtil.Configurations.CARD_CONFIG_NO_BRAND
 import com.worldpay.access.checkout.testutils.CardNumberUtil.PARTIAL_VISA
 import com.worldpay.access.checkout.testutils.CardNumberUtil.VISA_PAN
 import org.junit.Assert.assertEquals
-import org.junit.Before
 import org.junit.Test
 
-class CVVValidatorImplTest {
+class CVVValidatorTest {
 
-    private lateinit var cvvValidator: CVVValidator
-
-    @Before
-    fun setup() {
-       cvvValidator = CVVValidatorImpl(null)
-    }
-
-    @Test
-    fun `given an empty cvv and null card configuration then should be only completely and partially invalid`() {
-        assertEquals(Pair(ValidationResult(partial = false, complete = false), null), cvvValidator.validate("", null))
-    }
+    private var cvvValidator: CVVValidator = CVVValidator()
 
     @Test
     fun `given a empty cvv and non-null card configuration then should be only partially valid`() {
```

Comment:
I suppose tests that assert on partially valid are here only temporarily until we remove that partial of the ValidationResult?

Reply:
yes that is right